.github/workflows/tests.yml

```diff
@@ -46,15 +46,15 @@ jobs:
       - name: Install NuGet
         uses: nuget/setup-nuget@v1
         with:
-          nuget-version: 5.x
+          nuget-version: 6.x
 
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
 
       - name: Setup Python virtual environment
-        run: pip install virtualenv && python -m virtualenv venv
+        run: python -m pip install virtualenv && python -m virtualenv venv
```

Comment:
It looks like this workflow oesn't need python or pip, so we can remove this.

Reply:
The tests here use python